src/figuresClasses.ts

```diff
@@ -1,19 +1,77 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
```

Comment:
```suggestion
  color: Color;
```
Consider creating a type or an enum for the color